### DISCUSSION TITLE
Add new settings for long cursor trail

### DISCUSSION THREAD

**Discussion by crumpledmint** [ppy/osu#19644] [Ideas]
As for now the only way to customize cursor is to change it's look in the skin and it's size in game. With addition of cursormiddle in the skin we're able to make long trail but it lacks customization. It would be cool to have an option to control density of the trail (number of individual cursortrail.png's in the whole trail) and it's length

**Comment by Siiiuuuuu** [comment]
Bumping this. I started a discussion recently without seeing this, my idea was to make it an adjustable slider. I believe this would be a change requiring minimal effort and would be much appreciated 🙏
